test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -131,6 +131,15 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<echo>DUMP_OPTION: ${DUMP_OPTION}</echo>
 		<echo>EXTRADUMPOPT: ${EXTRADUMPOPT}</echo>
 
+		<!-- Use -Xint to make the startuphint viewable on the cache-->
+		<exec executable="${JAVA_COMMAND}" failonerror="true">
+			<arg line="-Xscmx16m -Xshareclasses:name=ddrextjunitSCC -Xcompressedrefs -Xgcpolicy:gencon" />
+			<arg line="-Xint" />
+			<arg value="-cp" />
+			<arg value="${TEST_RESROOT}/DDR_Test.jar" />
+			<arg value="j9vm.test.corehelper.SCCTestHelper" />
```

Comment:
Can you use j9vm.test.corehelper.CoreGen here so that you don't need to add SCCTestHelper.java ?

Reply:
I tried to use  the j9vm.test.corehelper.CoreGen and it caused the other tests failed. Hence, I added the  SCCTestHelper.java.